Release/src/websockets/client/ws_client_wspp.cpp

```diff
@@ -337,7 +337,7 @@ class wspp_callback_client : public websocket_client_callback_impl,
 
                     m_external_message_handler(incoming_msg);
                 }
-                return true;
+                return m_config.is_auto_pong_enabled();
```

Comment:
[RFC 6455 Section  5.5.2](https://tools.ietf.org/html/rfc6455#section-5.5.2) basically says that you MUST send a pong in answer to a ping. Section 5.5.3 says that if you receive multiple pings before you get a chance to send a pong, you MAY only acknowledge the last one. However supporting that behaviour would be more complicated than just having a Boolean in config.

Therefore I suggest _not_ adding `is_auto_pong_enabled` in this PR.

Reply:
Yeah but we need to return true or false here, and before we didn’t acknowledge the ping by a pong at all as we had no callback yet; so we need to add it

Reply:
I have found that websocketpp sends pong in response to ping by default, and will do so if you just return `true`, so I would stick with doing that, as you had originally.